diff --git a/Package.swift b/Package.swift
--- a/Package.swift
+++ b/Package.swift
@@ -10,7 +10,7 @@
             targets: ["SignInWithApple"])
     ],
     dependencies: [
-        .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", from: "7.0.0")
+        .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", "7.0.0"..<"9.0.0")
     ],
     targets: [
         .target(
